source/Mission.cpp

```diff
@@ -892,13 +905,19 @@ string Mission::BlockedMessage(const PlayerInfo &player)
 	if(flagship)
 		subs["<ship>"] = flagship->Name();
 
+	const auto &playerConditions = player.Conditions();
+	bool toAcceptPassed = toAccept.IsEmpty() || toAccept.Test(playerConditions);
+	subs["<conditions>"] = toAcceptPassed ? "meet" : "do not meet";
```

Comment:
An empty condition set returns true when tested, so we can simplify things here.
```suggestion
	subs["<conditions>"] = toAccept.Test(playerConditions) ? "meet" : "do not meet";
```
What is the purpose of this substitution, though? How would you expect this to be used?

Reply:
“You need another 2 bunks, and do meet the requirement to have a jump drive install.”

“You have enough space, and you do meet the requirement to have a jump drive installed.”